ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/[slug]/PauseButton.tsx

```diff
@@ -135,7 +136,7 @@ export default function PauseFunctionButton({ functionSlug, disabled }: PauseFun
     return null;
   }
 
-  const isPaused = !fn.current && !fn.archivedAt;
+  const { isPaused } = fn;
```

Comment:
I deleted the comment by mistake 🤦‍♀️ We have two places with the same isPaused logic, I'm not very familiar with this but worth checking them:
https://github.com/inngest/inngest/blob/main/ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/%5BenvironmentSlug%5D/functions/%5Bslug%5D/layout.tsx#L40

https://github.com/inngest/inngest/blob/main/ui/apps/dashboard/src/queries/functions.ts#L119

Reply:
@goodoldneon I am late following up to this; did you take care of these too?